membersrvc/ca/eca.go

```diff
@@ -314,8 +369,17 @@ func (ecap *ECAP) CreateCertificatePair(ctx context.Context, in *pb.ECertCreateR
 		} else {
 			obcECKey = ecap.eca.obcPub
 		}
+		if role == 1 {
```

Comment:
See https://github.com/hyperledger/fabric/pull/1444. Using just an int makes the code difficult to read. Use the role type instead.